src/styles/sidebar/components/tag-editor.scss

```diff
@@ -0,0 +1,45 @@
+@use "../../mixins/forms";
+@use "../../mixins/buttons";
+@use "../../variables" as var;
+
+.tag-editor {
+  &__input {
+    &::-webkit-calendar-picker-indicator {
+      // Turn off the default arrow icon
+      display: none;
+    }
+    @include forms.form-input;
+    width: 100%;
+    &:focus {
+      @include forms.form-input-focus;
+    }
+  }
+
+  &__tag-list {
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  &__tag-item {
+    display: flex;
+    margin-right: 0.5em;
+    margin-bottom: 0.5em;
+  }
+
+  &__edit {
+    color: var.$grey-6;
+    background: var.$grey-1;
+    border: 1px solid var.$grey-3;
+    border-radius: 2px 0 0 2px;
+    border-right-width: 0;
+    padding: 0 0.5em;
+  }
+
+  &__delete {
+    @include buttons.action-button;
```

Comment:
I'm not sure about re-using this mixin here because the other style properties need to fit with it. There is a possibility that someone could change the `action-button` mixin and not realize that it is used for tags as well as buttons.

Reply:
I think we can remove action button, but Id still like to base this off the base button mixin because after all, it is a button